h/services/group_json_presentation.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from h.presenters.group_json import GroupJSONPresenter, GroupsJSONPresenter
+
+
+class GroupJSONPresentationService(object):
+    def __init__(self, session, request_authority, route_url):
+        self._session = session
+        self._request_authority = request_authority
+        self._route_url = route_url
+
+    def present(self, group):
+        presenter = GroupJSONPresenter(group, self.get_links)
+        return presenter.asdict()
+
+    def present_all(self, groups):
+        presenter = GroupsJSONPresenter(groups, self.get_links)
+        return presenter.asdicts()
+
+    def get_links(self, group):
```

Comment:
This would be more well-factored and consistent as its own service similar to the existing `links` service (that one is for annotations specifically) but this PR is already too big, etc.